Frameworks/BlockchainCertificates/Requests/CertificateValidationRequest.swift

```diff
@@ -368,6 +486,28 @@ public class CertificateValidationRequest : CommonRequest {
     }
 }
 
+/// This is a simplified adaptation of the _getDataToHash function from
+/// https://github.com/digitalbazaar/jsonld-signatures. 
+/// This function prefixes the normalized json-ld with optional headers. Blockcerts only uses the 
+/// created date header for now.
+private func _getDataToHash(input: String, date: String) -> String {
+    let toHash = "http://purl.org/dc/elements/1.1/created: " + date + "\n" + input
+    return toHash
+}
+
+
+private  func hexStringFromData(input: NSData) -> String {
```

Comment:
This looks a lot like my extension `toHex` on `String`. I think you can use that instead of this.